src/main/java/org/edumips64/ui/swing/DineroFrontend.java

```diff
@@ -46,8 +46,15 @@
   // one instance of DineroFrame will be created in EduMIPS64
   private static final Logger logger = Logger.getLogger(DineroFrontend.class.getName());
   private static JTextField path, params;
-  private static JButton execute;
+  private static JButton execute, configure, create;	//configure and create buttons added to create panels and cache parameters
   private static JTextArea result;
+  private static JLabel pathLabel, paramsLabel, cacheLeveLabel, cacheTypeLabel;	//labels for cache level and cache type added
+  private static int argLevel;	//static declaration of array Level argument
+	private static char argType;	//static declaration of array type argument
+	private static Box cachePanel;	//static container globally declared for flexibility
+	private static JScrollPane scrollPane;	//static delclaration of scrollbar
+  public static JPanel panelL1, panelL2, panelL3, panelL4, panelL5;	//static panels to add configuration components
+  private static JComboBox cacheLevel, cacheType;		//static delcaration of cache combo boxes
```

Comment:
`JComboBox` is a generic class (https://docs.oracle.com/javase/8/docs/api/javax/swing/JComboBox.html). Please use the generic version (in this case, `JComboBox<String>`).

Reply:
The changes have been made. Thank you.

Reply:
I don't see the change.. did you push your changes?